src/__tests__/containers/HomePageComponent.test.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import { mount } from 'enzyme';
+import configureMockStore from 'redux-mock-store';
+import Slider from 'react-animated-slider';
+import { Chip } from 'react-materialize';
+import HomePageComponent from '../../components/HomePageComponent';
+
+const mock = configureMockStore();
+
+const store = mock({
+  article: {},
+});
+
+const props = {
+  fetchAll: () => {},
+  articles: {
+    results: [{
+      title: 'title',
+      description: 'description',
+      body: 'body',
+      slug: 'test-slug',
+      image_url: 'url',
+      tags: ['test'],
+      author: {
+        username: 'test',
+        image_url: 'https://placehold.it/20x20',
+      },
+    }],
+  },
+};
+const wrapper = mount(<HomePageComponent {...props} store={store} />);
+
+describe('renders <HomePageComponent/>', () => {
+  it('tests if component did mount', () => {
+    const spy = jest.spyOn(HomePageComponent.prototype, 'componentDidMount');
+    wrapper.instance().componentDidMount();
+    expect(spy).toHaveBeenCalled();
+  });
+  it('tests render <Slider/>', () => {
```

Comment:
Have a blank line to separate each `it` block

Reply:
Noted 👍